### DISCUSSION TITLE
EmberZNet Zigbee Router 8.0.x firmware for Silicon Labs

### DISCUSSION THREAD

**Discussion by Hedda** [darkxst/silabs-firmware-builder#141] [General]
FYI, @Nerivec has posted experimental updated (8.0.2) **Zigbee Router firmware** for the Sonoff ZBDongle-E here:

* https://github.com/Nerivec/silabs-firmware-builder/releases/tag/v2024.6.2-update1

Quote:

- Add experimental support for Sonoff ZBDongle-E Router firmware.
  - The adapter should automatically try to join a nearby open network every 10 seconds as long as it is not already joined (will continue indefinitely as long as it has power). This should remove the need to use the physical buttons (take apart the casing).
    - The LED will turn on and off (slowly) to alert of the fact the adapter is running a new scan. Once joined, the LED will stay fixed.
  - The RST button (in the casing) has 4 distinct behaviors:
      a.   While joined, a click will trigger permit join (broadcast) for 60 seconds.
      b.   **While joined, a press for over 5 seconds force the adapter to leave the network.**
      c.   While not joined, a click will run a scan and try to join a nearby open network. Should not be needed since it should run automatically on schedule.
      d.   **While not joined, a press for over 5 seconds will factory reset the adapter.**
  - Compatible with existing Zigbee2MQTT converter. *With updated software version info so you can tell which were updated (if you have several). You may need to re-interview for the updated info to show up.*
  - Flashing can be done via [Ember ZLI](https://github.com/Nerivec/ember-zli/wiki)

**Comment by Hedda** [comment]
@darkxst FYi, they are working on upstream now here -> https://github.com/NabuCasa/silabs-firmware-builder/pull/85

* https://github.com/NabuCasa/silabs-firmware-builder/pull/85

**Comment by darkxst** [comment]
@Hedda, yes i am aware, I started this discussion (under my other username). I will be publishing router builds here soon.
